public/data/Ratheshan03.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Ratheshan Sathiyamoorthy",
+    "bio": "BSc (Hons) Computer Science Undergraduate | Full Stack Software developer | Open Source Enthusiast",
+    "avatar": "https://github.com/Ratheshan03.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Ratheshan03",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/ratheshan-sathiyamoorthy-3aa2891b9/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Ratheshan_03",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/ratheshan_03/",
+        "icon": "instagram"
+      }
+    ],
+    "milestones": [
+        {
+          "title": "Started Freelancing",
+          "date": "June 2021",
+          "icon": "dollar",
+          "color": "grey",
+          "description": "Started freelancing",
+          "url": "https://www.fiverr.com/ratheshan03?up_rollout=true"
+        }
+      ]
+  }
```

Comment:
```suggestion
}

```

Reply:
Okay noted ! I'll make the changes asap.

Reply:
By the way all parentheses are closed right? I'm confused could you explain what is happening here..

Reply:
Yes, they are closed. This suggestion is to fix the indentation and add an endline. 🙂

Reply:
@Panquesito7 

Yeah sure I'll make the changes asap! 😃 Thankyou for notifying.